docs/reference.md

```diff
@@ -0,0 +1,217 @@
+- **path**: the path of the file or directory
+- **recursive** (default: True): operate on files and directories recursively
+
+
+## ls(path, all=True, human_readable=True, size=True, list=True)
+
+Run `ls` command on the remote server.
+
+
+##### Arguments
+
+- **path**: the path of the file or directory to consider
+- **all** (default: True): do not ignore entries starting with `.`
+- **human_readable** (default: True): print sizes in a humand readable format
+- **size** (default: True): also print size
+- **list** (default: True): use a long listing format
+
+
+## mv(src, dest)
+
+Run the `mv` command on the remote server.
+
+##### Arguments
+
+- **srv**: the path for the source
+- **dest**: the path of the destination
+
+
+## cp(src, dest, interactive=False, recursive=True, link=False, update=False)
+
+Run the `cp` command on the remote server.
+
+##### Arguments
+
+- **srv**: the path for the source
+- **dest**: the path of the destination
+- **interactive** (default: False): prompt before overwrite
+- **recursive** (default: True): copy directories recursively
+- **link** (default: False): hard link files instead of copying
+- **update** (default: False): copy only when the SOURCE file is newer than the
+  destination file or when the destination file is missing
+
+# File helpers
+
+
+## put(local, remote, force=False)
+
+Send a local file or directory to the remote server.
+
+##### Arguments
+
+- **local**: a reference to a file (can be a `pathlib.Path` instance, a `str`
+  or a `io.BytesIO` instance) or to a directory (`pathlib.Path` or `str`)
+- **remote**: the remote path
+- **force** (default: False): override remote file even if it's newer
+
+
+## get(remote, local)
+
+Fetch a remote file.
+
+##### Arguments
+
+- **local**: a reference to a file (can be a `pathlib.Path` instance, a `str`
+  or a `io.BytesIO` instance)
+- **remote**: the remote path
+
+
+# Context managers
+
+## sudo(set_home=True, preserve_env=True, user=None, login=None)
+
+Run the command in a `sudo` context. See `man sudo` for more info about using
+`sudo`.
+
+##### Arguments
+
+- **set_home** (default: True): request that the security policy set the HOME
+  environment variable to the home directory specified by the target user's
+  password database entry
+- **preserve_env** (default: True): preserve environment variable
+- **user** (default: None): run the command as this user
+- **login** (default: None): run the shell specified by the target user's
+  password database entry as a login shell
+
+
+## cd(path)
+
+Prefix all path to be run in command with this path.
+
+##### Arguments
+
+- **path** (default: `~`): the path to use as prefix
+
+
+## env(**kwargs)
+
+Define environment variables.
+
+##### Arguments
+
+- **kwargs**: key/value pair to set environment variables
+
+
+## screen(name)
+
+Run the command inside a screen.
+
+##### Arguments
+
+- **name** (default: `usine`) the name of the screen to be created
```

Comment:
Here you are formatting the default value as code, which you weren't previously (like with `True`)